src/bottom-sheet/bottom-sheet.ts

```diff
@@ -0,0 +1,210 @@
+import { html, LitElement, PropertyValues, nothing } from 'lit';
```

Comment:
I need some events to bubble up to Standard DOM (composed:true) which I can understand bottom sheet is opened, closed, dismissed, and any possible other events.